wiki/Manual.md

```diff
@@ -1534,6 +1545,42 @@ print(dir(ccxt.hitbtc()))           # Python
 var_dump (new \ccxt\okcoinusd ()); // PHP
 ```
 
+## Accounts
```

Comment:
Why would we put the description of `fetchAccounts` into the intro section though?